From c87f7d045eb56685078b206f9078fb8b7fa5e3a5 Mon Sep 17 00:00:00 2001
From: Hyundo Lee <54927573+hdlee27@users.noreply.github.com>
Date: Thu, 13 Aug 2020 13:30:42 +0900
Subject: [PATCH 1/1] Use __asm__ instead of asm for c99 build

Signed-off-by: Hyundo Lee <54927573+hdlee27@users.noreply.github.com>
---
 components/soc/esp32/include/soc/cpu.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/components/soc/esp32/include/soc/cpu.h b/components/soc/esp32/include/soc/cpu.h
index f28feb5..247bce8 100644
--- a/components/soc/esp32/include/soc/cpu.h
+++ b/components/soc/esp32/include/soc/cpu.h
@@ -21,6 +21,8 @@
 #include "xtensa/corebits.h"
 #include "xtensa/config/core.h"
 
+#define asm __asm__
+
 /* C macros for xtensa special register read/write/exchange */
 
 #define RSR(reg, curval)  asm volatile ("rsr %0, " #reg : "=r" (curval));
-- 
2.7.4

