From 307e2e3481efe7f0c00a5b518aef3ac678793cab Mon Sep 17 00:00:00 2001
From: Hyundo Lee <54927573+hdlee27@users.noreply.github.com>
Date: Wed, 4 Nov 2020 19:48:12 +0900
Subject: [PATCH 1/1] add CONFIG_STDK_IOT_CORE_EASYSETUP_HTTP_USE_SOCKET_API

Signed-off-by: Hyundo Lee <54927573+hdlee27@users.noreply.github.com>
---
 makefiles/config.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/makefiles/config.mk b/makefiles/config.mk
index 13011a2..10fe9d0 100644
--- a/makefiles/config.mk
+++ b/makefiles/config.mk
@@ -12,6 +12,11 @@ CONFIG_STDK_IOT_CORE_BSP_SUPPORT_EMW3166=y
 CONFIG_STDK_IOT_CORE_OS_SUPPORT_FREERTOS=y
 CONFIG_SAMSUNG_BUILD_ENG=y
 CONFIG_STDK_IOT_CORE_LOG_LEVEL_DEBUG=y
+CONFIG_STDK_IOT_CORE_EASYSETUP_HTTP_USE_SOCKET_API=y
+
+ifeq ($(CONFIG_STDK_IOT_CORE_EASYSETUP_HTTP_USE_SOCKET_API), y)
+STDK_CFLAGS += -DCONFIG_STDK_IOT_CORE_EASYSETUP_HTTP_USE_SOCKET_API
+endif
 
 ifeq ($(CONFIG_STDK_IOT_CORE),y)
 STDK_CFLAGS += -DCONFIG_STDK_IOT_CORE
-- 
2.7.4

