From f48f79b620738350a6420f8f0de6f7e7b6928227 Mon Sep 17 00:00:00 2001
From: Hyundo Lee <54927573+hdlee27@users.noreply.github.com>
Date: Wed, 4 Nov 2020 18:30:03 +0900
Subject: [PATCH 1/1] add CONFIG_STDK_IOT_CORE_EASYSETUP_HTTP_USE_SOCKET_API

Signed-off-by: Hyundo Lee <54927573+hdlee27@users.noreply.github.com>
---
 MiCO/net/stdk/config.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/MiCO/net/stdk/config.mk b/MiCO/net/stdk/config.mk
index 26a7b28..c3e1f98 100644
--- a/MiCO/net/stdk/config.mk
+++ b/MiCO/net/stdk/config.mk
@@ -11,6 +11,11 @@ CONFIG_LWIP_USE_HTTPD_FOR_EASYSETUP=y
 CONFIG_STDK_IOT_CORE_BSP_SUPPORT_EMW3080=y
 CONFIG_STDK_IOT_CORE_OS_SUPPORT_MOCOS=y
 CONFIG_SAMSUNG_BUILD_ENG=
+CONFIG_STDK_IOT_CORE_EASYSETUP_HTTP_USE_SOCKET_API=y
+
+ifeq ($(CONFIG_STDK_IOT_CORE_EASYSETUP_HTTP_USE_SOCKET_API), y)
+STDK_CFLAGS += -DCONFIG_STDK_IOT_CORE_EASYSETUP_HTTP_USE_SOCKET_API
+endif
 
 ifeq ($(CONFIG_STDK_IOT_CORE),y)
 STDK_CFLAGS += -DCONFIG_STDK_IOT_CORE
-- 
2.7.4

